### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by ashrayjain** [yujinakayama/atom-auto-update-packages#17]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/Applications/Atom.app/Contents/Resources/app/src/atom.js:54:11)
Object.activate (/Users/ashrayjain/.atom/packages/auto-update-packages/lib/auto-update-packages.coffee:21:9)
```

**Comment by aioue** [comment]
Duplicate of #16.

**Comment by yujinakayama** [comment]
Fixed in auto-update-packages 1.0.0. (#31)
